internal/artifacts/module_arch_test.go

```diff
@@ -466,6 +466,12 @@ module-types:
 			Ω(err).Should(Succeed())
 		})
 	})
+
+	var _ = Describe("isArchive", func() {
```

Comment:
Can you add tests for the other use cases (folder, folder with "x.zip" name, zip file, war file, jar file, other non-archive file - maybe txt or without an extension)?
Also maybe we should add .gz to the possible list of archive files (not sure if in this CL)

Reply:
tests added. I didn't add .gz (because you are mot sure..)